ntbs-service/Models/Enums/AlertStatus.cs

```diff
@@ -0,0 +1,7 @@
+namespace ntbs_service.Models.Enums 
+{
+    public enum AlertStatus {

```

Comment:
C# convention (atleast wiht MS) to have opening braces on new lines (here and in AlertType enum too